README.md

```diff
@@ -1,96 +1,100 @@
 # Storage Extension Specification
 
 - **Title:** Storage
-- **Identifier:** <https://stac-extensions.github.io/storage/v1.0.0/schema.json>
+- **Identifier:** <https://stac-extensions.github.io/storage/v2.0.0/schema.json>
 - **Field Name Prefix:** storage
-- **Scope:** Item, Collection
+- **Scope:** Item, Catalog, Collection
```

Comment:
Should `Asset` be included since it is checked under the "Fields" section?

Reply:
This refers to the standalone specifications in stac-spec, which are Catalog, Item and Collection.